src/components/Loading/Loading.tsx

```diff
@@ -108,6 +108,7 @@ const Loading: FC = () => {
                 onClick={() => {
                   disconnect();
                   close();
+                  location.reload();
```

Comment:
Shouldn't it be `window.location.reload()` ? Approving anyway as i don't see anything breaking here.
```suggestion
                  window.location.reload();
```

Reply:
nice... using github to commit this suggestion is nice